test/Parse/strange_interpolation.swift

```diff
@@ -1,5 +1,8 @@
 // RUN: %target-typecheck-verify-swift -swift-version 4.2
-// RUN: %target-run-simple-swift -swift-version 4.2 %s | %FileCheck %s
+// RUN: %target-build-swift -swift-version 4.2 %s -o %t
+// RUN: %target-run %t | %FileCheck %s
```

Comment:
@brentdax I tweaked this as well because `%target-run-simple-swift` passes its arguments to the *executable,* not to the compiler. At the moment the test suite's still being run in pre-Swift-5 mode anyway so it didn't change anything.

Reply:
I'm fine with the test being moved—I didn't realize we were trying to keep that directory non-executable. Do you want me to do that?